openkfw/TruBudget
# 2005 env joi validation

### Checklist




### Description


### How to test

1. Validate new generated docs for env vars
2. Incorrectly set variables and try to start TruBudget - it should fail
3. When setting env vars incorrectly the proper error message should be displayed in descriptive way
4. Try copy of `env_example` file to `.env`. App should start directly or only the essential variables should be missing.




Closes #2005
